### ISSUE TITLE
Clarify whether accesslint-ci does also work with other CIs (Travis, GitLab)

### ISSUE THREAD

**Issue by dusek** [accesslint/accesslint-ci#15]
Hi,

the README leaves me in doubt whether this project requires to be run on Circle CI, or whether it will also run on e.g. Travis CI with some reduced functionality (i.e. it will not comment on GitHub in case of accessibility violations, but will still successfully fail the build in such case, enabling a person to see the build failed and look at the build log to find the specific accessibility violations)?

The bug is that IMHO the README should make this clear, as there are surely many people out there who use other CIs than Circle CI.

Thanks.

**Comment by DirtyF** [comment]
Examples on other CIs would be so cool!

**Comment by ckundo** [comment]
**tl;dr: accesslint only runs on CircleCI right now, TravisCI is queued up in our backlog!**

Thanks @dusek for the note! I see the ambiguity and will clarify here:

accesslint-ci relies on CircleCI's artifacts and will fail to run if it cannot find those or if a `CIRCLE_TOKEN` is missing from the environment variables.

I love the idea of degrading in other CI environments; I'll make that an interim step to full TravisCI support.

**Comment by ckundo** [comment]
I updated the GitHub project description to mention CircleCI specifically; I'm going to leave the README as is for now, since it also specifies CircleCI in a couple places, and since I'd like to keep it more generic with the understanding that the end goal is to support multiple CI's.

**Comment by IschaGast** [comment]
Love to see multiple ci support especially Travis. Keep up the super duper a11y works 🤘
